applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -341,6 +341,19 @@ void KratosGeoMechanicsApplication::Register() {
     KRATOS_REGISTER_VARIABLE( DT_TEMPERATURE )
     KRATOS_REGISTER_VARIABLE( NORMAL_HEAT_FLUX )
 
+    KRATOS_REGISTER_VARIABLE( AIR_TEMPERATURE )
+    KRATOS_REGISTER_VARIABLE( SOLAR_RADIATION )
+    KRATOS_REGISTER_VARIABLE( AIR_HUMIDITY )
+    KRATOS_REGISTER_VARIABLE( PRECIPITATION )
+    KRATOS_REGISTER_VARIABLE( WIND_SPEED )
+    KRATOS_REGISTER_VARIABLE( A1_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( A2_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( A3_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( ALPHA_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( QF_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( SMIN_COEFFICIENT )
+    KRATOS_REGISTER_VARIABLE( SMAX_COEFFICIENT )
```

Comment:
Just a general question: do we need to mind these names as well (to not run into the same issue as with the duplicate element names?)

Reply:
As discussed with the team, a comment is added above the variables to indicate they are for Micro-climate, and for now they will stay as they are now until further discussions with CIMNE.